settings.gradle

```diff
@@ -11,10 +11,23 @@ pluginManagement {
         maven {
             name = 'Sponge Snapshots'
             url = 'https://repo.spongepowered.org/repository/maven-public/'
+        }
+        maven {
+            name = 'Sleeping Town'
+            url = 'https://repo.sleeping.town/'
+            content {
+                includeGroup 'agency.highlysuspect'
+                includeGroup 'agency.highlysuspect.minivan'
```

Comment:
are both these needed?

Reply:
First one is for the plugin itself (group `agency.highlysuspect`, artifact `minivan`), second one is for [the plugin marker artifact](https://repo.sleeping.town/agency/highlysuspect/minivan/agency.highlysuspect.minivan.gradle.plugin/) (group `agency.highlysuspect.minivan`, artifact `agency.highlysuspect.minivan.gradle.plugin`). [Weird gradle thing](https://docs.gradle.org/current/userguide/plugins.html#sec:plugin_markers).